pom.xml

```diff
@@ -316,6 +316,32 @@ THE SOFTWARE.
               <overWriteSnapshots>true</overWriteSnapshots>
             </configuration>
           </execution>
+          <execution>
+            <id>copy-fips-libs</id>
+            <goals>
+              <goal>copy</goal>
+            </goals>
+            <configuration>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>org.bouncycastle</groupId>
+                  <artifactId>bc-fips</artifactId>
+                  <version>1.0.2.4</version>
```

Comment:
discussion from https://github.com/jenkinsci/jenkins-test-harness/pull/825#discussion_r1743366773

Reply:
@jtnord do you mean when Jenkins will have a class called FIPS200? :)

Reply:
> you may want a TODO here, as this is not the latest version of bc-fips but for a valid reason.

the latest bc-fips 1.x is `1.0.2.5`  but it should not be updated as it is not yet certified.
there is also a bc-fips `2.0.0.0` which is certified but at FIPS 140-3 (rather than 140-2) which is the current target of the JEP.